### DISCUSSION TITLE
List of Zim Plugins

### DISCUSSION THREAD

**Discussion by vossenjp** [zim-desktop-wiki/zim-desktop-wiki#1882] [General]
I just learned about The [pastefromhtml](https://github.com/cetinkaya/pastefromhtml) plugin from https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/1879#issuecomment-1019532070 and it sounds great!  But it's not listed in https://github.com/jaap-karssenberg/zim-wiki/wiki/Plugins that i can see, so is that a documentation bug?  What else is missing?  What's the canonical list of Zim plugins?

**Comment by jaap-karssenberg** [comment]
PLease add it - the list is a wiki page that can be edited. THese are all
3rd party plugins, so we rely on 3rd parties to add them to the list as
well.

**Comment by vossenjp** [comment]
Added.  Anyone else who reads this and knows of other plugins...please add those.

**Reply by introt** [reply]
Did you check if it still works with the latest version? @dford16 from #1898 might find it useful

**Comment by vossenjp** [comment]
Yes, in zim-0.74.3-t202201271844~ubuntu*.  I have not tested extensively, but "Tools > Paste from HTML" does paste in more-or-less Zim code from a web/browser source.  Interestingly it is not rendered, so you have to hit CTRL-R or cut it then re-paste (assuming you also have "Tools > Prefs > Editing: Reformat on the fly" set).  That's all "good enough" for anything I might want right now,

**Reply by introt** [reply]
Great, thank you!

**Comment by introt** [comment]
The list could do with a makeover to filter out the outdated plugins. Some of them have stuff like "doesn't support Zim 0.66 or above", but imo version compatibility should be a "first-class" feature.

Any opinions on the preferred way for sorting? I'd go for subheadings for versions, but should entries be moved or duplicated? There's also already the "tags" subheading, which would need to be duplicated.

A table could also work, but is way easier to accidentally mess up, and may be harder to read. There's also the problem of some plugin versions supporting a specific Zim version, while others have some backwards compatibility.

Suggested format:

### Third-party plugins

#### Zim 0.74 `(assuming point releases don't break API)`

* Plugin name/link: short description

##### Tags (an example grouping)

* Tag plugin name/link: short description

#### Zim 0.73

_These plugins haven't been tested on the latest release. If found to work, they should be moved to the upper list; otherwise they should be moved to the legacy plugin list below._

#### Legacy plugins

_If a version isn't specified, the plugin has not been tested since this list was reformatted._

* Legacy plugin name/link for Zim _versions here_: short description
